<ui version="4.0" >
 <author>Xiantia</author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>UIdebuger</class>
 <widget class="QFrame" name="UIdebuger" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>205</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>13</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking" >
   <bool>true</bool>
  </property>
  <property name="windowTitle" >
   <string>Debuger</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>13</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>150</width>
         <height>16</height>
        </size>
       </property>
       <property name="title" >
        <string>Remote debugger</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="butonNextStepOver" >
            <property name="text" >
             <string>Next step over</string>
            </property>
            <property name="icon" >
             <iconset resource="../resources/resources.qrc" >:/Icons/Icons/editgoto.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="butonNextStepInto" >
            <property name="text" >
             <string>Next step into</string>
            </property>
            <property name="icon" >
             <iconset resource="../resources/resources.qrc" >:/Icons/Icons/editgoto.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="butonContinue" >
            <property name="text" >
             <string>Continue</string>
            </property>
            <property name="icon" >
             <iconset resource="../resources/resources.qrc" >:/Icons/Icons/editredo.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="gdbWorking" >
            <property name="autoFillBackground" >
             <bool>false</bool>
            </property>
            <property name="frameShape" >
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="lineWidth" >
             <number>1</number>
            </property>
            <property name="text" >
             <string>Debugger no started </string>
            </property>
            <property name="scaledContents" >
             <bool>false</bool>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>13</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <widget class="QWidget" name="page_2" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>1</hsizetype>
          <vsizetype>13</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../ide/src/resources/resources.qrc" />
  <include location="../resources/resources.qrc" />
 </resources>
 <connections/>
</ui>
